go/chat/push.go

```diff
@@ -622,11 +622,6 @@ func (g *PushHandler) Activity(ctx context.Context, m gregor.OutOfBandMessage) (
 			if conv, err = g.G().InboxSource.Expunge(ctx, uid, nm.InboxVers, nm.ConvID, nm.Expunge, nm.MaxMsgs); err != nil {
 				g.Debug(ctx, "chat activity: unable to update inbox: %s", err.Error())
 			}
-			activity = chat1.NewChatActivityWithExpunge(chat1.ExpungeInfo{
-				ConvID:  nm.ConvID,
-				Expunge: nm.Expunge,
-				Conv:    g.presentUIItem(conv),
-			})
```

Comment:
Deleting this will lead to line 637 sending a pointer to a default-valued ChatActivity. That's probably not good.

How about making `activity` a pointer and then changing all the `activity = chat1...` to `activity = &chat1...` in this function? Then on 637 make it only call `notifyNewChatActivity` is `activity != nil`.